### ISSUE TITLE
Label rotation does not work for row when labelAngle = 0

### ISSUE THREAD

**Issue by chuanconggao** [vega/vega-lite#1664]
Please:
- [X] Check for duplicate issues
- [X] Describe how to reproduce the bug /  the goal of the new feature request
Can be reproduced using the online editor:
```json
{
    "data": {"values": [
      {"a": "A","b": 28, "c": "A"}, 
      {"a": "B","b": 55, "c": "A"}, 
      {"a": "C","b": 43, "c": "A"}
    ]},
    "mark": "bar",
    "encoding": {
        "x": {
            "field": "b", "type": "quantitative", "axis": false
        },
        "y": {
            "field": "a", "type": "nominal", "axis": false
        },
        "row": {
            "field": "c", "type": "nominal",
            "axis": {
                "title": "", "labelAngle": 0
            }
        }
    }
}
```
- [X] Provide an example spec and/or screenshots (if applicable) 
When labelAngle is not set, we have:
![vega 1](https://cloud.githubusercontent.com/assets/6963729/19976352/a38d9f94-a1ac-11e6-98da-0f3e7a083469.png)
However, when labelAngle = 0, there is no difference from default:
![vega 1](https://cloud.githubusercontent.com/assets/6963729/19976352/a38d9f94-a1ac-11e6-98da-0f3e7a083469.png)
Further, when labelAngle = 360, it works:
![vega 2](https://cloud.githubusercontent.com/assets/6963729/19976369/b0f2f0f8-a1ac-11e6-8618-3e5e1a70c0bf.png)

It seems labelAngle = 0 is treated the same as default. How can I force it without using 360?

**Comment by kanitw** [comment]
Thanks for reporting.  This is definitely a bug.  

Since you can get around using 360, I guess we can punt on this and fix it as a part of 2.0 release?  
If you have pressing need for a patch release, let us know.  I can try to fix and make a 1.x patch release.  Thank you.

**Comment by chuanconggao** [comment]
Using 360 works for me right now. So surely there is no need to rush it. 😄 Really looking forward to 2.0.
